src/ESM.t.sol

```diff
@@ -296,4 +296,34 @@ contract ESMTest is DSTest {
 
         usr.callJoin(esm, 10);
     }
+
+    function test_file_new_min() public {
+        esm = new ESM(address(gem), address(end), address(this), 10);
```

Comment:
I would use more realistic values for these tests, such as 75 * WAD

Edit: I just realized that would be inconsistent with the rest of the test suite, so I don’t know if it makes sense. Maybe update the whole test suite ? lol